docker-compose.dev.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  website:
+    build: .
+    ports:
+      - 8080:8080
+    volumes:
+      - .:/app
+    env_file:
+      - .env
+    depends_on:
+      - postgres
+    tty: true
+    command: ./wait-for-postgres.sh python runserver.py
+  postgres:
+    image: postgres:10.4
+    restart: always
```

Comment:
I'm not sure about the `restart: always`. I think this is going to restart with the dev machine or as soon as Docker service is up which may not be suitable for development.